demosplan/DemosPlanCoreBundle/ResourceTypes/OrgaResourceType.php

```diff
@@ -216,6 +219,14 @@ protected function getProperties(): array
                 ->readable(false, $this->getAllowedRoles(...));
         }
 
+        if ($this->currentUser->hasPermission('feature_orga_new_procedure_edit')) {
+            $properties[] = $this->createAttribute($this->canCreateProcedures)->readable(true, function (Orga $orga): bool {
+                $currentCustomer = $this->currentCustomerService->getCurrentCustomer();
+
+                return $this->accessControlPermissionService->canCreateProcedure($orga, $currentCustomer, RoleInterface::PRIVATE_PLANNING_AGENCY);
```

Comment:
can we avoid the role check here somehow? Roles should only be used in Permissions.php